iso15118/secc/states/din_spec_states.py

```diff
@@ -587,9 +568,8 @@ async def process_message(
             )
             return
 
-        self.comm_session.evse_controller.ev_data_context.ev_session_context.soc = (
-            precharge_req.dc_ev_status.ev_ress_soc
-        )
+        ev_data_context = self.comm_session.evse_controller.ev_data_context
+        ev_data_context.update_pre_charge(precharge_req)
```

Comment:
```suggestion
        ev_data_context.update_pre_charge_parameters(precharge_req)
```

Reply:
it was updated, closing for a fresh review